guides/source/security.md

```diff
@@ -1214,6 +1214,44 @@ Rails.application.config.content_security_policy do |policy|
 end
 ```
 
+##### Using the `Report-To` and `Reporting-Endpoints` Headers to send violation reports
+
+INFO: _These headers are part of the **Experimental** [`ReportingAPI`][], and therefore not widely supported by some browsers_
+
+The [`Report-To`][] and [`Reporting-Endpoints`][] headers are used to specify a group of endpoints, for which browsers should send reports related to certain types of issues or policy violations like CSP Violations, Crash and Deprecation Reports.
+
+When present, `report-to` overrides the directive `report-uri`.
+
+Though the `report-to` directive is intended to replace the deprecated `report-uri` directive, `report-to` isn’t supported in most browsers yet. So for compatibility with current browsers while also adding forward compatibility when browsers get `report-to` support, you can specify both `report-uri` and `report-to`.
+
+
+Use the [`report-to`][] directive to send reports to a designated URI.
+
+```ruby
+Rails.application.config.content_security_policy do |policy|
+  policy.report_to "/csp-violation-report-endpoint"
+end
+```
+
+Alternatively, use lambdas to define multiple endpoints that share the same configuration under a single group name.
+
+```ruby
+Rails.application.config.content_security_policy do |policy|
+  policy.report_to "default",  -> { {
+          default: {
+            urls: ["/csp-violation-report-endpoint", "https://example.com/csp-violation-report"],
+            max_age: 30.minutes,
+            include_subdomains: true
+          },
+
+          group_2: "https://example.com/hpkp-reports"
+    }
+  }
+end
```

Comment:
```suggestion
Rails.application.config.content_security_policy do |policy|
  policy.report_to "default",  -> do
    {
      default: {
        urls: ["/csp-violation-report-endpoint", "https://example.com/csp-violation-report"],
        max_age: 30.minutes,
        include_subdomains: true
      },

      group_2: "https://example.com/hpkp-reports"
    }
  end
end
```

I've used the `do/end` form here, instead of like I proposed above. I'm not sure which is better, I'm mainly focused on the indentation and alignment of braces here.